src/types/chart/chart.ts

```diff
@@ -57,6 +57,7 @@ export interface ExcelChartDefinition {
   readonly verticalAxisPosition: VerticalAxisPosition;
   readonly legendPosition: LegendPosition;
   readonly stacked?: boolean;
+  readonly aggregated?: boolean;
```

Comment:
if we can't export aggregated charts, `ExcelChartDefinition` should not have the `aggregated` key, right?